### DISCUSSION TITLE
[2021.10.18] 루시와 엘라 찾기

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#141] [SQL]
[루시와 엘라 찾기](https://programmers.co.kr/learn/courses/30/lessons/59046)

**Comment by hyk339** [comment]
[루시와 엘라 찾기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59046.sql)

**Comment by pyunji** [comment]
[루시와 엘라 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59046.sql)

**Comment by chaeyeon4u** [comment]
[루시와 엘라 찾기](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59046.sql)

**Comment by leechang4** [comment]
[루시와 엘라 찾기](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059046.sql)
